### ISSUE TITLE
Popup flickers on Mac

### ISSUE THREAD

**Issue by GoogleCodeExporter** [max99x/pagemon-chrome-ext#88]
```
What steps will reproduce the problem?

Click the extension icon on the toolbar when there is an updated website to 
view.

What is the expected output? What do you see instead?

Solid window quickly appears and able to select actions. Window slowly stutters 
and flickers; impossible to select any of the actions.

What version of the Chrome are you using? On what operating system?

8.0.552.208. Mac OS X 10.5.8.

Please provide any additional information below.

```

Original issue reported on code.google.com by `arylc...@gmail.com` on 19 Nov 2010 at 10:37

**Comment by GoogleCodeExporter** [comment]
```
Also, when there are "no changes found," the pop-up is perfectly fine (i.e. 
solid, with all actions clickable).
```

Original comment by `arylc...@gmail.com` on 19 Nov 2010 at 10:42

**Comment by GoogleCodeExporter** [comment]
```
Nevermind the previous comment. I just encountered the same flickering issue 
when there are "no changes found."
```

Original comment by `arylc...@gmail.com` on 20 Nov 2010 at 7:12

**Comment by GoogleCodeExporter** [comment]
```
Unfortunately I don't have access to a Mac machine to test this and cannot 
reproduce it on Windows or Linux. If you can trigger the bug on another OS, 
please provide instructions. If this no longer applies on the current version 
of Chrome, I would also appreciate a confirmation.
```

Original comment by `max99x` on 15 Feb 2011 at 1:35
* Added labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I don't have easy access to a non-Mac machine, but I will try to find another 
way to reproduce the issue for you. It is still happening on the most current 
Chrome version I am using - 9.0.597.102 beta.
```

Original comment by `arylc...@gmail.com` on 15 Feb 2011 at 1:55

**Comment by GoogleCodeExporter** [comment]
```
I am attaching two .mp4 (H.264) video recordings demonstrating the issue. I 
hope it helps you narrow down the problem. 

The pop-up turning opaque at the end of "Issue.mp4" is an "optical illusion": 
because it's a video, all motion stops in the last frame (which is also why it 
is impossible to take a screenshot of the issue).

To stop the flickering, I have to keep clicking the flickering area until I 
randomly hit the button that opens a tab showing changes to the updated page. 
Once the tab with the changes opens, the pop-up becomes opaque again.
```

Original comment by `arylc...@gmail.com` on 16 Feb 2011 at 4:58

Attachments:
* [Issue.mp4](https://storage.googleapis.com/google-code-attachments/pagemon-chrome-ext/issue-88/comment-5/Issue.mp4)
* [Normal Behavior.mp4](https://storage.googleapis.com/google-code-attachments/pagemon-chrome-ext/issue-88/comment-5/Normal Behavior.mp4)

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the video! It seems the cause is that extension popups on Mac appear 
gradually rather than at once like on Windows/Linux and it's conflicting with 
some window size measurement that PM is doing. I'll see if I can find the exact 
cause and a solution.
```

Original comment by `max99x` on 16 Feb 2011 at 5:05

**Comment by GoogleCodeExporter** [comment]
```
Could you please check if disabling animations from the advanced settings in 
the PM options page affects the issue in any way?
```

Original comment by `max99x` on 16 Feb 2011 at 11:34

**Comment by GoogleCodeExporter** [comment]
```
That was one of the first things I tried. Unfortunately, disabling animation 
seems to have no effect at all. In fact, the videos I sent you before were 
taken with animation disabled. Is it possible that the animation option is 
non-functional on Mac OS X? And if it is, should I submit a separate bug report 
for that issue?
```

Original comment by `arylc...@gmail.com` on 17 Feb 2011 at 3:09

**Comment by GoogleCodeExporter** [comment]
```
Please check if this still happens in v3.0.3, which I've just released to the 
gallery. You might need to force an update from Chrome's extensions page. I've 
enforced the popup sizes now so it shouldn't try to dynamically resize and redo 
the layout which is what I guess is the cause of that flickering.
```

Original comment by `max99x` on 17 Feb 2011 at 3:43
* Changed title: **Popup flickers on Mac**

**Comment by GoogleCodeExporter** [comment]
```
Flickering is fixed in 3.0.3. Thanks! On a side note, I tried the animation 
setting again, and I am now noticing a difference with on vs. off--not sure if 
it's new version, or simply my failure to notice the animation in the pop-up 
before.
```

Original comment by `arylc...@gmail.com` on 17 Feb 2011 at 3:53

**Comment by GoogleCodeExporter** [comment]
```
Good to hear. As for animation, there shouldn't be any difference from the last 
version (at least on OSs that didn't have the flickering issue).
```

Original comment by `max99x` on 17 Feb 2011 at 3:56
* Changed state: **Fixed**
